test/unit/machine-learning/machine-learning.spec.ts

```diff
@@ -162,9 +162,9 @@ describe('MachineLearning', () => {
     });
 
     it('should throw given invalid credential', () => {
-      const expectedError = 'Failed to initialize ML client with the available ' +
-          'credential. Must initialize the SDK with a certificate credential or application default ' +
-          'credentials to use Firebase ML API.';
+      const expectedError = 'Failed to initialize Google Cloud Storage client with ' +
```

Comment:
Once you defer calling `admin.storage()`, I'd expect this error to not occur until `createModel()` is called.

Reply:
done